File: pb/src/main/java/org/sqlbuilder/pb/WordResolver.java
package org.sqlbuilder.pb;

import org.sqlbuilder.common.DeSerialize;
import org.sqlbuilder.common.Resolver;

import java.io.File;
import java.io.IOException;

public class WordResolver extends Resolver<String,Integer>
{
	public WordResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: pb/src/main/java/org/sqlbuilder/pb/WordResolver.kt
package org.sqlbuilder.pb

import org.sqlbuilder.common.DeSerialize
import org.sqlbuilder.common.Resolver
import java.io.File

typealias PbWordResolvable = String
typealias PbWordResolved = Int

class WordResolver(ser: String) : Resolver<PbWordResolvable, PbWordResolved>(DeSerialize.deserialize<Map<PbWordResolvable, PbWordResolved>>(File(ser)))
